library/packages/src/modules/PackagesUI.rb

```diff
@@ -299,6 +300,8 @@ def RunPackageSelector(options)
 
       widget_options = Builtins.add(widget_options, :confirmUnsupported) if !display_support_status.nil? && display_support_status
 
+      widget_options = Builtins.add(widget_options, :onlineSearch) if options.fetch("enable_online_search", false)
```

Comment:
Why do we need `fetch` here? Would just a simple `if options["enable_online_search"]` be enough?